Framework/TestHelpers/src/InstrumentCreationHelper.cpp

```diff
@@ -81,10 +81,7 @@ void addFullInstrumentToWorkspace(MatrixWorkspace &workspace,
   instrument->markAsSource(source);
 
   // Define a sample as a simple sphere
```

Comment:
This comment may not be accurate any more?

Reply:
Yes you're right. I've just updated it. Thanks for ploughing through this one!